fem/pgridfunc.cpp

```diff
@@ -299,6 +299,108 @@ const
    return (DofVal * LocVec);
 }
 
+double ParGridFunction::GetValue(ElementTransformation &T,
+                                 const IntegrationPoint &ip,
+                                 int comp, Vector *tr) const
```

Comment:
Should not this replace the existing ParGridFunction::GetValue implementation? The old one is not prepared for the `ElementTransformation` types and does not support integral FEs.